src/hotspot/share/ci/ciMethod.cpp

```diff
@@ -1131,6 +1133,21 @@ int ciMethod::instructions_size() {
   return _instructions_size;
 }
 
+int ciMethod::inline_instructions_size() {
+  if (_inline_instructions_size == -1) {
+    GUARDED_VM_ENTRY(
+                     CompiledMethod* code = get_Method()->code();
+                     if (code != NULL && (code->comp_level() == CompLevel_full_optimization)) {
+                       int isize = code->insts_end() - code->verified_entry_point() - code->post_call_nop_size();
+                       _inline_instructions_size = isize > 0 ? isize : 0;
+                     } else {
+                       _inline_instructions_size = 0;
+                     }
+                     );
```

Comment:
Indentation looks weird. I see that you copied it from `ciMethod::instructions_size`, that one should be fixed as well.

Reply:
ok